src/Tizen.NUI/src/public/Images/ImageLoading.cs

```diff
@@ -23,9 +23,7 @@ namespace Tizen.NUI
     /// <summary>
     /// Loading an image.
```

Comment:
maybe better sth like "A set of tools providing image loading", "A class containing methods to load an image from file ..."